packages/paste-core/primitives/box/src/types.ts

```diff
@@ -91,7 +91,7 @@ export interface BoxElementProps extends Omit<React.HTMLAttributes<HTMLElement>,
   /**
    * Same as HTML
    *
-   * @type {string}
+   * @type {boolean}
```

Comment:
question: hmmm I'm not so sure about the changes on lines 94 and 129.

Reply:
+1

Reply:
oh seems like he was just updating the description to match the actual type. I'll keep